client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkShuffleUtils.java

```diff
@@ -45,9 +41,9 @@
 import com.tencent.rss.client.factory.CoordinatorClientFactory;
 import com.tencent.rss.storage.util.StorageType;
 
-public class RssShuffleUtils {
+public class RssSparkShuffleUtils {
```

Comment:
The name refactor should be included in another PR.
Too many code in spark module are updated, but this PR is target to MR's reduce part.

Reply:
This is because that the original RssShuffleUtils contains some spark-related dependency,  which should not be used by MR.  So this PR has to move it from spark, and put it into common.
Otherwise, MR needs to the dependency of Spark, it looks more strange.